src/backend/test.rs

```diff
@@ -105,6 +108,12 @@ impl TestBackend {
     }
 }
 
+impl Display for TestBackend {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", buffer_view(&self.buffer))
+    }
+}
+
```

Comment:
I think you're right. `impl Display` is the right choice.

Do you think we should move buffer_view within this method? It seems unnecessary other than for display purposes. Happy to approve either way.

Reply:
> Do you think we should move buffer_view within this method? It seems unnecessary other than for display purposes. Happy to approve either way.

Currently, it is still used by `assert_buffer()`, but once #161 lands , I think the assertion macro can be used in its stead. 
And then moving the `buffer_view()` will make sense IMO.

Reply:
Ah yes, you’re right. The use in the assert method precludes moving it. So nothing to do there.